### ISSUE TITLE
Cannot change base file or template from one repository to another

### ISSUE THREAD

**Issue by comradekingu** [WeblateOrg/weblate#4535]
On https://hosted.weblate.org/settings/strongbox/translations/#vcs I accidentally set it up to point to
https://github.com/strongbox-password-safe/Strongbox at first

Then it was changed to https://github.com/strongbox-password-safe/babel
where the structure is different, so the filemask and monolingual base file failed, and it couldn't be changed to

**Filemask**: StrongBox/*.lproj/Localizable.strings
and
**Template for new translations**: StrongBox/Localizable.strings

Basically it can't be set up like https://hosted.weblate.org/settings/strongbox/strongbox/#files
without recreating the component.

Not sure if the StrongBox component should use template, basefile for monolingual or both, but that is another matter(?)

"de" was started as a translation in the translation component before trying to switch it over, if that makes any difference.

**Comment by nijel** [comment]
Why it could not be changed? Did you get some error?

**Comment by comradekingu** [comment]
https://hosted.weblate.org/settings/strongbox/translations/#files

I did it now and got
"File does not exist" using "StrongBox/Localizable.strings" in "monolingual base file.
"Base file for new translations not found." if I put it in "Template for new translations"

You can try it. The component is just there for debugging at this point, and the "StrongBox" one is the one used.

It isn't the end of the world, but it helps to have a bug-report that someone can find I thought.

**Comment by nijel** [comment]
The problem is that these repositories do not have common history. In that case, changing the repo URL is not enough as that just leads to a merge error, but you have to do repository reset manually (in https://hosted.weblate.org/projects/strongbox/translations/#repository).

I'm not sure what's the best approach here. Weblate could just reset to the new upstream repo, but that would lead to losing anything done so far, that's why it's currently not done automatically.

**Comment by nijel** [comment]
I will add a warning about this situation to the settings page, so that user can be aware of this.
